pecos/xmc/xlinear/predict.py

```diff
@@ -103,6 +103,15 @@ def parse_arguments():
         metavar="THREADS",
         help="number of threads to use (default -1 to denote all the CPUs)",
     )
+
+    parser.add_argument(
+        "-so",
+        "--select-output",
+        type=str,
+        default=None,
+        metavar="PATH",
+        help="path to the npz file of the select output matrix (CSR, nr_insts * nr_labels)",
```

Comment:
Add: `only-topk` and `beam-size` are ignored if given.

Reply:
Note has been added to help.